pkg/registry/maven/maven_test.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2025 Google LLC
+// SPDX-License-Identifier: Apache-2.0
+
+package maven
+
+import (
+	"context"
+	"testing"
+)
+
+func TestReleaseURL(t *testing.T) {
+	r := HTTPRegistry{}
+	url, err := r.ReleaseURL(context.Background(), "com.google.guava:guava", "33.4.8-jre", TypePOM)
```

Comment:
would you mind making this a parameterized test (i.e. loop through a list of test cases)? Even if it's just 1 or 2 examples, it makes it a lot easier to incrementally add new cases.